cloudslang_cli.rst

```diff
@@ -137,21 +137,27 @@ extensions. If multiple input files are being used and they contain an
 input with the same name, the input in the file that is loaded last will
 overwrite the others with the same name.
 
+Input files that contain ``inputs`` in their name can be loaded
```

Comment:
having `inputs`in the file name in not enforced currently, however it s best practice to use `<name>.inputs.yaml` format (the file still need to be YAML)